### DISCUSSION TITLE
Line break for long text  for fixed column width

### DISCUSSION THREAD

**Discussion by ashek1412** [Power-Components/livewire-powergrid#1107] [Q&A]
Hi,
I cannot implement line break for column with fixed width. all text are displayed in single line.
is there any way to break the text in multiple lines

![image](https://github.com/Power-Components/livewire-powergrid/assets/18645594/cfced2b6-af63-45e5-ae95-ef6e205bde50)

**Comment by abhaypithadiya** [comment]
Hello! You can add a class like "text-wrap" for Bootstrap, "break-all" for Tailwind or add a style in your bodyAttribute. Something like this.

```
    public function columns(): array
    {
        return [
            Column::make('Title', 'title')
                ->bodyAttribute('text-wrap')
                ->searchable(),
        ];
    }
```

**Reply by ashek1412** [reply]
I tried but its doesn't work

**Reply by luanfreitasdev** [reply]
Hello @ashek1412 , try this:

    public function columns(): array
    {
        return [
            Column::make('Title', 'title')
                ->bodyAttribute('!text-wrap') // <--- add "!"
                ->searchable(),
        ];
    }

**Reply by ashek1412** [reply]
I am using tailwind . I want fixed column width with line break.  I tried   !break-all but still same output

**Reply by ashek1412** [reply]
any update ?

**Reply by abhaypithadiya** [reply]
Try using this:

```
    public function columns(): array
    {
        return [
            Column::make('Title', 'title')
                ->headerAttribute('', 'width:10%;')
                ->searchable(),
        ];
    }
```

**Reply by ashek1412** [reply]
nothing happened

**Reply by abhaypithadiya** [reply]
Try adding a different percentage? Try 5%?

**Reply by ashek1412** [reply]
![image](https://github.com/Power-Components/livewire-powergrid/assets/18645594/a693f1a1-559e-4cec-be4d-923e35c22cf9)

still same

**Reply by abhaypithadiya** [reply]
Is the title going to be this long without any spaces in between? or is this just a dummy text?

**Reply by ashek1412** [reply]
Title has spaces

**Reply by luanfreitasdev** [reply]
I uploaded a fix and it will be available in v5.0.4

https://github.com/Power-Components/livewire-powergrid/pull/1148

```php
Column::add()
       ->contentClasses('!whitespace-normal') // <------- add whitespace-normal
       // ->sortable(),
```
